autogpt/agent/agent.py

```diff
@@ -281,21 +275,23 @@ def start_interaction_loop(self):
                     if not plugin.can_handle_post_command():
                         continue
                     result = plugin.post_command(command_name, result)
-                if self.next_action_count > 0:
-                    self.next_action_count -= 1
-
-            # Check if there's a result from the command append it to the message
-            # history
-            if result is not None:
-                self.full_message_history.append(create_chat_message("system", result))
-                logger.typewriter_log("SYSTEM: ", Fore.YELLOW, result)
-            else:
-                self.full_message_history.append(
-                    create_chat_message("system", "Unable to execute command")
-                )
-                logger.typewriter_log(
-                    "SYSTEM: ", Fore.YELLOW, "Unable to execute command"
-                )
+                if self.autonomous_cycles_remaining > 0:
+                    self.autonomous_cycles_remaining -= 1
+
+                # Check if there's a result from the command append it to the message
```

Comment:
There is something really wrong here, that code should be outside of the else scope, why was it tabbed?